query/artifacts.go

```diff
@@ -0,0 +1,21 @@
+package query
+
+import (
+	"time"
+
+	"github.com/flanksource/duty/context"
+	"github.com/flanksource/duty/models"
+	"github.com/google/uuid"
+)
+
+func ArtifactsByCheck(ctx context.Context, checkID uuid.UUID, checkTime time.Time) ([]models.Artifact, error) {
```

Comment:
Can we do these as views, so that they are ready for use by the UI ?  We would not return the connectionId and server side path

Reply:
This is just a WHERE clause on the artifacts table. Do we need a view for this?